src/main/treeDataProviders/dependenciesTree/goDependenciesTreeNode.ts

```diff
@@ -0,0 +1,20 @@
+import { DependenciesTreeNode } from './dependenciesTreeNode';
+import { IComponentMetadata } from '../../goCenterClient/model/ComponentMetadata';
+export class GoDependenciesTreeNode extends DependenciesTreeNode {
+    private _goCenter!: IComponentMetadata;
```

Comment:
Put it inside the constructor. Lets call it `componentMetadata` or `goCenterComponentMetadata`. Please update the getter as well.
```suggestion
```

Reply:
Changed to `componentMetadata`